unidist/core/backends/mpi/core/communication.py

```diff
@@ -4,22 +4,26 @@
 
 """MPI communication interfaces."""
 
+import time
 import mpi4py
-mpi4py.rc(recv_mprobe=False)
 from mpi4py import MPI
 
-import time
-
 from unidist.core.backends.mpi.core.serialization import MPISerializer
 
+mpi4py.rc(recv_mprobe=False)
```

Comment:
@no-ponomarev please, take a look on this. Does this affect MPI execution? (I've tested on census - this works well)

Reply:
This call should be done before any MPI initializations, therefore moving this call may affect corner cases (which I don't remember now) and should be undo.

Reply:
Ok. Ignoring comment was added for imports after `mpi4py.rc(recv_mprobe=False)` to make `flake8` check passed.

Reply:
Thanks! Not sure how to configure mpi4py outside this import code, maybe add a TODO to rework this later

Reply:
Agree with message. `TODO` comment was added